src/Validation/validateInput.js

```diff
@@ -0,0 +1,10 @@
+export default function validateInput(inputMail, inputPassword) {
```

Comment:
Amrita, for now, you have created this validation folder directly inside src. Later on, we can move it to the utils folder. If you are not aware of it yet it's okay we can discuss it later.